### ISSUE TITLE
Two sway sessions, two behaviors for keyboard shortcut

### ISSUE THREAD

**Issue by PaulDebus** [swaywm/sway#1610]
Hey,

I am starting to really doubt myself. I wanted to define a  shortcut: Alt-p (or M-p in emacs), but it did not work. It just did not trigger the command. I tried both emacs and a self developed application, but neither would react to pressing Alt-p. All other shortcuts work just fine.

I then tried an i3 session, where everything worked fine. Using the same config in another sway session also worked fine. What really made be doubt myself was, that starting a second sway session on tty2 with the exact same configuration as the first one, all the shortcuts worked. I can reproduce the issue after a reboot of the system. 

TLDR: I have two sway sessions using the same config file on tty1 and tty2, tty2 works fine, tty1 too, except for the Alt-p shortcut, which does not work in any application.

Thank you in advance for your help, let me know if you need any more information. Attached you find the config file in use.

[config.txt](https://github.com/swaywm/sway/files/1763880/config.txt)

**Comment by SirCmpwn** [comment]
Can you point out the specific bindsym command that does not work?

**Comment by PaulDebus** [comment]
It is not a sway bindsym it is an emacs keybinding or a shortcut in a program. I do not use the Alt key at all in may sway config

**Comment by SirCmpwn** [comment]
Can I get a debug log?

**Comment by PaulDebus** [comment]
[sway_working_log.txt](https://github.com/swaywm/sway/files/1763966/sway_working_log.txt)
[sway_not_working_log.txt](https://github.com/swaywm/sway/files/1763967/sway_not_working_log.txt)

**Comment by PaulDebus** [comment]
I tried a little more and if I put e.g.
```
bindysm Mod1+p exec notify-send "alt-p"
```
into my sway config, it works well. Then, I tried different small test programs to find it out. A simple ncurses python skript did work, whereas a small Qt application (see [gist](https://gist.github.com/PaulDebus/45c99ec86b7f5d95d83af9de8f78b70e)) did not react to alt-p but to alt-o and ctrl-p

I do not know if this helps anything....
